docs/src/pages/docs/rag/vector-databases.mdx

```diff
@@ -190,6 +190,8 @@ The dimension size must match the output dimension of your chosen embedding mode
 - OpenAI text-embedding-3-large: 3072 dimensions
 - Cohere embed-multilingual-v3: 1024 dimensions
 
+> **Important**: Index dimensions cannot be changed after creation. To use a different model, delete and recreate the index with the new dimension size.
```

Comment:
In memory I put the dimension size in the index name to get around this `memory_index_${dimensions}`
Could be good to add a note about doing that